java/client/src/main/java/glide/api/models/commands/RangeOptions.java

```diff
@@ -10,9 +10,10 @@
 import lombok.RequiredArgsConstructor;
 
 /**
- * Arguments for {@link SortedSetBaseCommands#zrange} and {@link
- * SortedSetBaseCommands#zrangeWithScores}
+ * Arguments for {@link SortedSetBaseCommands#zcount}, {@link SortedSetBaseCommands#zrange} and
```

Comment:
```suggestion
 * Arguments for {@link SortedSetBaseCommands#zcount}, {@link SortedSetBaseCommands#zrange}, and
```
https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/